src/main/java/org/opensearch/knn/indices/ModelMetadata.java

```diff
@@ -304,13 +323,14 @@ public static ModelMetadata fromString(String modelMetadataString) {
         String[] modelMetadataArray = modelMetadataString.split(DELIMITER, -1);
         int length = modelMetadataArray.length;
 
-        if (length < 7 || length > 10) {
+        if (length < 7 || length > 11) {
```

Comment:
Are we able to get any more information around the model version based on what is stored in model metadata? Itd be useful to know what models are created after 2.14

Reply:
For anything going forward, we should be able to, but anything before 2.17 is going to have an empty version